fastlane/Fastfile

```diff
@@ -298,6 +298,52 @@ platform :ios do
     trigger_release_build(branch_to_build: "release/#{version}")
   end
 
+  #####################################################################################
+  # download_localized_strings_and_metadata
+  # -----------------------------------------------------------------------------------
+  # Downloads localized app strings and App Store Connect metadata from GlotPress.
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane download_localized_strings_and_metadata
+  #####################################################################################
+  desc 'Downloads localized metadata for App Store Connect from GlotPress'
+  lane :download_localized_strings_and_metadata do |options|
+    # WordPress: leverage the default release toolkit setup.
+    # It's not clear in the action name, but this downloads both `.strings`
+    # translations for the app and App Store Connect metadata.
+    ios_update_metadata(options)
+    # Jetpack: use the custom setup. Notice that we don't need extra steps to
+    # download `.strings` because Jetpack uses the same codebase as WordPress.
+    download_jetpack_localized_app_store_metadata
+  end
+
+  #####################################################################################
+  # update_metadata_on_app_store_connect
+  # -----------------------------------------------------------------------------------
+  # Uploads the localized metadata tracked in the project repository to App Store
+  # Connect, optionally updating screenshots, too.
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane update_metadata_on_app_store_connect [with_screenshots:<with screenshots>]
+  #
+  # Example:
+  # bundle exec fastlane update_metadata_on_app_store_connect
+  # bundle exec fastlane update_metadata_on_app_store_connect with_screenshots:true
+  #####################################################################################
+  desc 'Updates the App Store Connect localized metadata'
+  lane :update_metadata_on_app_store_connect do |options|
```

Comment:
We hinted at having a lane replacing the call to `deliver` in this PR.

The reason I think it's valuable to use a lane is that we could remove the `Deliverfile` and have the `Fastfile` as the single source of truth. What do you think?

I went down that path for the Jetpack logic, but kept the current setup for WordPress – there's enough going on in here already 😄 

I'm happy to revert to use `deliver` if you don't think this is a good approach, maybe passing a dedicated `Jetpack-Deliverfile` to the call. But I'd ask to do it as a followup PR against the next release since this one is getting close to the release finalization date.

Reply:
I agree it can make sense to have a single source of truth. But:

 - Having that PR ending up doing it one way for Jetpack and another for WordPress actually moves away from that goal, because now we have 3 potential source of truth (`Deliver` + a bit in `Fastfile` too for WP, and `Jetpack-Fastfile` for Jetpack. So personally I would revert the change in that PR to make it only call `deliver` for now – aligning this with the current setup used for WordPress – and work on this on a separate PR
 - That separate PR thwt would work on having a unique source of truth might lead to a debate (e.g. given `Jetpack-Fastfile` is already a separate file, should we put the source of truth in `Fastfile`, making it not obvious when reading the `Jetpack-Fastfile` that some constants come from the main `Fastfile` but are not obivous and visible in the `Jetpack-Fastfile`? Or given we have 2 fastfiles, should we actually consider the `Deliverfile` as a good thing to be used as a **common** source of truth for both?

If we only had one Fastfile I'd agree to get rid of the `Deliverfile` and use the `Fastfile` as the single source of truth, but given we already split it the fastfile there… I think we should brainstorm what's best and keep that debate for a separate PR